home/models.py

```diff
@@ -295,7 +295,29 @@ def clean(self, value):
                 f"{self.MEDIA_CAPTION_MAX_LENGTH} characters, your message is "
                 f"{len(result['message'])} characters long"
             )
+
+        # find variables
+        vars_in_msg = re.findall(r"{{(.*?)}}", result["message"])
+        non_digit_variables = [var for var in vars_in_msg if not var.isdigit()]
+        if non_digit_variables != []:
+            errors["message"] = ValidationError("Please enter numeric variables only.")
```

Comment:
Since we've got a list of the variables that are non-numeric, `non_digit_variables`, maybe we can include that in the error message to make it easier for the user to find out what we're referring to. We can probably do something similar with the other error message as well.